README.md

```diff
@@ -1,38 +1,36 @@
-# The Swift Community Podcast
+<p align=center><img src="https://github.com/SwiftCommunityPodcast/podcast/blob/master/Resources/Logo%402x.png" width="250"/></p>
 
-## About
+### 🎙 For the community. By the community.
 
-<img src="https://github.com/SwiftCommunityPodcast/podcast/blob/master/Resources/Logo%402x.png" width="250"/>
+Welcome, we're glad you're here! The Swift Community Podcast is a community-driven show focusing on the people that build, use, and support Swift, Apple's open source programming language. This repository is a place where anyone who is interested can stay informed about the latest developments, learn new techniques and engage with others in the community.
```

Comment:
Maybe change the last sentence to something like: `This repository contains the episodes, show notes, and supporting documentation that make of the podcast and which are used to help it run as a open source and community driven show.`

Otherwise, it makes it seem like this repo is for staying informed about the great Swift open source project.